external/akka/src/main/scala/org/apache/spark/streaming/akka/ActorReceiver.scala

```diff
@@ -90,6 +90,48 @@ object ActorReceiver {
 @DeveloperApi
 abstract class ActorReceiver extends Actor {
 
+  /**
+   * start a new actor-based receiver, spark streaming will asynchronously send the corresponding
+   * ActorRef object to the caller actor of this API
+   *
+   * @param props the props object of the receiver actor
+   * @param name  the name of the actor
+   */
+  def startNewActorReceiver(props: Props, name: String) {
```

Comment:
The reason I did not create directly callable APIs for these is that result is not received as a return value of the functions, instead it is received as a message asynchronously. This is not the usual behavior of the API methods and thus I was unsure about having them at all. @tdas thoughts ?